### PR TITLE
compat: use eval instead of ast.literal_eval on python 2.4

### PR DESCRIPTION
**Pull request by fennm**

### CODE REVIEW

**Comment by solj** [comment]
:+1:

File: doc/development/compat.txt [hunk 1]
```diff
@@ -129,6 +129,8 @@ below, since some of these implementations may be feature-incomplete.
 +----------------+--------------------------------+--------------------------------------------+
 | MutableMapping | :class:`UserDict.DictMixin`    | :class:`collections.MutableMapping` (2.6+) |
 +----------------+--------------------------------+--------------------------------------------+
+| literal_eval   | :func:`ast.literal_eval`       | :func:`eval`                               |
```
Comment (stpierre) [inline doc/development/compat.txt]: Shouldn't these be the other way around?

Comment (fennm) [inline doc/development/compat.txt]: Good catch.  I fixed that, and also changed the header to be more clear

**Comment by stpierre** [comment]
+1

**Comment by solj** [comment]
I think ast.literal_eval is "technically" only 2.6+.

**Comment by solj** [comment]
Merged, thanks!

**Comment by fennm** [comment]
Thanks!

**Pull request merged**